First preview (#20)
herflis/concepts-temp: content/preview2/content-management.md

---
title: "Content Management"
metaTitle: "sensenet - Content Management"
metaDescription: "sensenet Content Management"
---

# What does content management mean?

For content management we mean the processes of storing, editing, organizing and delivering content.

> # What is content regarding to sensenet?

Author: ```suggestion

# What is content regarding to sensenet?
```

In sensenet everything is a content. Documents, tasks, users, projects, even system configuration files – everything – are stored and managed in the [content repository](/concepts/content-repository). It is the foundation that makes content management easy and powerful at the same time. All content items are organized in a [content tree](/concepts/content-tree), so you can always think in subtrees and inheritance when managing and presenting content.

# Basic content management actions in sensenet

> All the basic content management actions are available in sensenet, so you can create a content, edit or delete it, copy or move it to another folder.

Author: ```suggestion
All the features that can be called basic content management actions are included in sensenet, so you can create, edit or delete a content, copy or move it to another folder.
```

To learn more go to [Actions](/concepts/actions)

# Can anybody do anything?

> Content management is a collaborative process where every user has its own part. Permission management is critical in such system so basically what the user can or cannot do with a certain content.

Author: ```suggestion
Content management is a collaborative process where every user has its own part (for example a user can have the permission to create a new content, he/she can open and see a content, but doesn’t have the permission to edit or delete it.). Permission management is critical in such system as we have to specify what the user can or cannot do with a certain content.
```

> Content management is a collaborative process and every user has his own part in it. At this point we can talk about roles and permissions. [role based permission, document level permission], so basically what the user can or cannot do with the certain content.

Reviewer: ```suggestion
Content management is a collaborative process where every user has its own part. Permission management is critical in such system so basically what the user can or cannot do with a certain content.
```

> For example a user can have the permission to create a new content, he/she can open and see a content, but doesn’t have the permission to edit or delete it. Since sensenet has a tree based hierarchical content structure it ensures permission inheritance, so it is enough to set a permission rule one time and it comes down in lower levels of the tree. This means, if you set a permission on a document library it will be automatically applied to subfolders and contents (document level permission). You can easily set a permission also for a group and this way control their accessibilty to predefined contents (role based permission).

Reviewer: ```suggestion
For example a user can have the permission to create a new content, he/she can open and see a content, but doesn’t have the permission to edit or delete it. Since sensenet has a tree based hierarchical content structure it ensures permission inheritance, so it is enough to set a permission rule one time and it comes down in lower levels of the tree. This means, if you set a permission on a document library it will be automatically applied to subfolders and contents ([document level permission](/concepts/document-level-permissions)). You can easily set a permission also for a group and this way control their accessibilty to predefined contents ([role based permission](/concepts/role-based-permissions)).
```

> For example a user can have the permission to create a new content, he/she can open and see a content, but doesn’t have the permission to edit or delete it. Since sensenet has a tree based hierarchical content structure it ensures permission inheritance, so it is enough to set a permission rule one time and it comes down in lower levels of the tree. This means, if you set a permission on a document library it will be automatically applied to subfolders and contents ([document level permission](/concepts/document-level-permissions)). You can easily set a permission also for a group and this way control their accessibilty to predefined contents ([role based permission](/concepts/role-based-permissions)).

Author: ```suggestion
Since sensenet content repository is a tree based hierarchical content structure it ensures permission inheritance, so it is enough to set a permission rule one time and it comes down in lower levels of the tree. This means, if you set a permission on a document library it will be automatically applied to subfolders and contents ([document level permission](/concepts/document-level-permissions)). You can easily set a permission also for a group and this way control the accessibility of predefined contents ([role based permission](/concepts/role-based-permissions)).
```
